lib/Dialect/FIRRTL/Transforms/BlackBoxReader.cpp

```diff
@@ -388,14 +390,16 @@ StringAttr BlackBoxReaderPass::loadFile(Operation *op, StringRef inputPath,
 }
 
 /// Determine the output file for some operation.
-std::pair<OutputFileAttr, Priority>
+std::tuple<StringAttr, Priority, bool>
```

Comment:
Beyond `pair` consider a struct for improved readability of types and accessors.